API/Startup.cs

```diff
@@ -151,20 +151,22 @@ public void ConfigureServices(IServiceCollection services)
                         }
                     });
                 o.IncludeXmlComments($"{AppDomain.CurrentDomain.BaseDirectory}{typeof(Startup).Namespace}.xml", true);
+                Uri authorizationUrl = Environment.IsDevelopment() ? new Uri(Config.IdentityServer.DevelopmentIdentityUrl + "/connect/authorize") : new Uri(Config.IdentityServer.IdentityUrl + "/connect/authorize");
```

Comment:
Not a fan of this rule, but we ban the use of the ternary operator in our [style guide](https://github.com/DigitalExcellence/dex-backend/wiki/Coding-guidelines#banned-operators). Maybe we could add a private helper method to build the `authorizationUrl`?

Reply:
You're right, thanks!